docs/markdown/snippets/d_absolute_path_includes.md

```diff
@@ -0,0 +1,4 @@
+## Deprecated use of absolute paths to D impdirs
```

Comment:
Should this be as "Removal of"

Reply:
I don't think I can actually land this patch now, since we've hit 1.0 and not removed it, I think we're stuck with it till 2.0 :(

Reply:
Sad times :/